integration_tests/src/numpy.sh

```diff
@@ -2,6 +2,17 @@
 
 set -e
 
+NO_ISOLATION=""
+SKIP_DEPS=""
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --no-isolation) NO_ISOLATION="--no-isolation"; shift ;;
+        --skip-dependency-check) SKIP_DEPS="--skip-dependency-check"; shift ;;
+        *) echo "Unknown parameter: $1"; exit 1 ;;
+    esac
+done
```

Comment:
Maybe we can just forward `$@` to `pyodide build`, instead of handling these parameters separately?

Reply:
Addressed in e38a704953522fcdc7366f612d52b4c4e77e798b.